ichk/check.py

```diff
@@ -356,7 +385,13 @@ def run(self):
         hiera = ancestors + [storage_resource[Resource.name]]
         self.hiera = ";".join(hiera)
 
-        for dirname, subdirs, filenames in os.walk(self.vault_path):
+        if self.root_collection is None:
+            path_to_walk = self.vault_path
+        else:
+            path_to_walk = self.root_collection.replace("/" + self.root[Resource.zone_name], self.vault_path)
```

Comment:
This may replace too much depending on the zone name and the names of collections within.
```suggestion
            path_to_walk = self.root_collection.replace("/" + self.root[Resource.zone_name], self.vault_path, 1)
```

Reply:
Thanks! Good catch.